onecodex/distance.py

```diff
@@ -1,6 +1,7 @@
 from onecodex.exceptions import OneCodexException
 from onecodex.taxonomy import TaxonomyMixin
 from onecodex.lib.enums import AlphaDiversityMetric, BetaDiversityMetric, Rank
+import warnings
```

Comment:
Minor style comment: we tend to place imports from the standard library at the top of the file, followed by third-party imports, followed by our own `onecodex` imports (usually each of these groups are separated by a blank line). For example:

```python
import warnings

from onecodex.exceptions import OneCodexException
from onecodex.taxonomy import TaxonomyMixin
from onecodex.lib.enums import AlphaDiversityMetric, BetaDiversityMetric, Rank
```

Reply:
Implemented.